config/scout.php

```diff
@@ -125,7 +125,7 @@
     | source search engine with minimal configuration. Below, you can state
     | the host and key information for your own Meilisearch installation.
     |
-    | See: https://docs.meilisearch.com/guides/advanced_guides/configuration.html
+    | See: https://www.meilisearch.com/docs/learn/configuration/instance_options#all-instance-options
```

Comment:
Even though thats the "new" link, it would be useful to have an additional link to https://www.meilisearch.com/docs/learn/configuration/settings below so one can immediately find the index settings as well